html-relative-links/about-me/about-me.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>About Me</title>
+  <style>
+    img {
+      width: 300px;
+    }
+  </style>
+</head>
+<body>
+  <ul>
+    <li><a href="../index.html">Home Page</a></li>
+    <li><a href="../about-me/about-me.html">About Me</a></li>
```

Comment:
On this line, you are exiting the current directory just to go right back into it. Instead, you can just state the name of the file you need directly since it's in the same directory this path is being built from.

Reply:
https://user-images.githubusercontent.com/72685552/136162721-f9223f66-6f23-44ed-8113-6d81cf7c90a4.mov

Thank you! Completed this for all three respective files.